packages/bot-web-ui/src/components/dashboard/dbot-tours/mobile-tours/__tests__/onboarding-tour.spec.tsx

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line simple-import-sort/imports
 import React from 'react';
 import { mockStore, StoreProvider } from '@deriv/stores';
```

Comment:
i believe you should add it on top of the line that you dont want to get affected.